### PR TITLE
stolostron/backlog#26917--RBAC stuff, to include using a verb for the…

### PR DESCRIPTION
**Pull request by swopebe**
… implementation task

### CODE REVIEW

File: access_control/rbac_implement_rhacm.adoc [hunk 1]
```diff
@@ -1,257 +1,19 @@
 [#rhacm-rbac-implement]
 = Implementing role-based access control
 
-RBAC is validated at the console level and at the API level. Actions in the console can be enabled or disabled based on user access role permissions. View the following sections for more information on RBAC for specific lifecycles in the product:
+{product} RBAC is validated at the console level and at the API level. Actions in the console can be enabled or disabled based on user access role permissions. 
+
+The {mce} is a prerequisite and the cluster lifecycle function of {product}. To manage RBAC for clusters with the {mce}, use the RBAC guidance from the cluster lifecycle xref:../multicluster_engine/about/mce_rbac.adoc/mce_rbac.adoc#mce-rbac[Role-based access control] documentation.
```
Comment (dockerymick) [inline access_control/rbac_implement_rhacm.adoc]: Replace {product} with {product-title-short}

**Review by dockerymick — APPROVED** [review]
Left a comment. Thanks for removing this information, looks good to me

File: access_control/rbac_implement_rhacm.adoc (hunk 1 above)
Comment (swopebe) [reply access_control/rbac_implement_rhacm.adoc]: I don't understand why we have three words in that conref to just complicate things. lol

Comment (dockerymick) [reply access_control/rbac_implement_rhacm.adoc]: We can update the variable name for 2.7 Refresh

**Pull request merged**